src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -384,26 +384,21 @@ private void Browser_CoreWebView2InitializationCompleted(object sender, CoreWebV
 
             try
             {
-                 this.browser.NavigateToString(libraryHTMLPage);
```

Comment:
I suggest you handle the `CoreWebView2InitializationCompletedEventArgs e` argument
```

            if (!e.IsSuccess)
            {
                if(e.InitializationException != null)
                {
                    LogToDynamoConsole(e.InitializationException.Message);
                }
                LogToDynamoConsole("LibraryViewExtension CoreWebView2 initialization failed ");
                return;
            }
            
            ```

Reply:
Also probably should check the references to the model inside LogToConsole 

```

        internal void LogToDynamoConsole(string message)
        {
            this.dynamoViewModel?.Model?.Logger?.Log(message);
        }
        
```